packages/ui-components/src/components/toggle-tip/toggle-tip.tsx

```diff
@@ -0,0 +1,150 @@
+import { autoPlacement, computePosition, ComputePositionConfig, offset, shift, arrow } from '@floating-ui/dom';
+import { Component, Host, h, Prop, Element, Listen, Event, EventEmitter } from '@stencil/core';
+import { isEmpty } from 'lodash-es';
+import { ETooltipPosition } from '../../utils/types/components';
+import { DEFAULT_AUTOPLACEMENT_CONFIG, DEFAULT_OFFSET, DEFAULT_SHIFT_CONFIG, DEFAULT_TOOLTIP_CONFIG } from '../tooltip/tooltip.config';
+import { getArrowElementPositionConfig, OFFSET_WITH_ARROW } from './config';
+import { IToggleTip, IToggleTipEvents } from './toggle-tip.types';
+
+/**
+ * @part toggle-tip-container - The toggle tip container.
+ * @part toggle-tip-slot-content - The toggle tip slot content.
+ */
+@Component({
+	tag: 'kv-toggle-tip',
+	styleUrl: 'toggle-tip.scss',
+	shadow: true
+})
+export class KvToggleTip implements IToggleTip, IToggleTipEvents {
+	/** @inheritdoc */
+	@Prop({ reflect: true }) text?: string;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) position?: ETooltipPosition;
+	/** @inheritdoc */
+	@Prop({ reflect: true }) allowedPositions?: ETooltipPosition[];
+	/** @inheritdoc */
+	@Prop({ reflect: true }) options?: Partial<ComputePositionConfig>;
```

Comment:
This is not being used.

Reply:
Yep, this was used on the  autoUpdate function but forgot to remove.

Reply:
Not sure if you should remove it or use it, but to keep this PR small let's keep it this way.

Reply:
Don't forget this.